src/schema/v2/me/conversation/eventConnection/hybridConnection/fetchHybridConnection.ts

```diff
@@ -0,0 +1,139 @@
+// interface PaginatedSourceInput<T> {
+//   source: string
+//   nodes: Array<T>
+// }
+
+import { Connection, ConnectionArguments } from "graphql-relay"
+import { sortBy } from "lodash"
+import { hybridConnectionFromArraySlice } from "./hybridConnectionFromArraySlice"
+import { NodeWMeta } from "./hybridConnectionFromArraySlice"
+import { HybridOffsets } from "./hybridOffsets"
+
+// some typical relay pagination args (maybe there is a type for this)
+type SortDirection = "ASC" | "DESC"
```

Comment:
Aren't we currently limited to only `DESC` atm?

Reply:
i don't think so, removed this limitation.